### ISSUE TITLE
Cannot resolve a Slate point from DOM point

### ISSUE THREAD

**Issue by ncqwer** [ianstormtaylor/slate#3723]
#### Do you want to request a _feature_ or report a _bug_?
bug
#### What's the current behavior?
Under the situation that if a normally `parent element`'s children are all `void element`,clicking the area(`contenteditable=false`) that belong to `parent element` and  multiple times will result in an error(`Cannot resolve a Slate point from DOM point: [object Text],17`)

here is the exmaple:
[example](https://codesandbox.io/s/cool-cache-xn6u5?file=/src/App.js)
![Jun-05-2020 17-20-30](https://user-images.githubusercontent.com/12671657/83859775-f94cbe80-a750-11ea-9d1a-b6d965864dfd.gif)
The bug doesn't happen each time. 

Slate: 0.58.1
Browser: Chrome 
OS: Mac 

#### What's the expected behavior?
No error occured!

**Comment by maebert** [comment]
Probably the same issue as #3421.

**TL;DR** every element that has `contentEditable={false}` should also have `style={{ userSelect: 'none' }}` to avoid this issue.

**Comment by williamtstanley** [comment]
I am getting this behaviour even with `user-select: none` in the styles

**Comment by gorpher** [comment]
> Probably the same issue as #3421.
> 
> **TL;DR** every element that has `contentEditable={false}` should also have `style={{ userSelect: 'none' }}` to avoid this issue.

good idea.
it's solved,thanks.
